Oqtane.Client/Modules/Controls/ActionDialog.razor

```diff
@@ -56,6 +71,12 @@
     [Parameter]
     public string Text { get; set; } // optional - defaults to Action if not specified
 
+    [Parameter]
+    public bool ShowTextBox { get; set; } = false; // optional - specifies if the TextBox should be displayed in the dialog - default is false
```

Comment:
I can't entirely agree with this because we might have multiple inputs, so it would be nice to have the body as a template